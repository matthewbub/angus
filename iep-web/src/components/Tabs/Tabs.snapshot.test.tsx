import { render } from '@testing-library/react'
import { Tabs } from './index'

it('renders Tabs component unchanged', () => {
  const { container } = render(<Tabs tabs={[
    { label: 'Tab 1', href: '#', current: true, as: 'a' },
    { label: 'Tab 2', onClick: () => { }, current: false, as: 'button' },
    { label: 'Tab 3', href: '#', current: false, as: 'a' },
  ]} />)
  expect(container).toMatchSnapshot()
})
