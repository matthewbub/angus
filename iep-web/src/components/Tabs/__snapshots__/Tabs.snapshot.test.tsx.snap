// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Tabs component unchanged 1`] = `
<div>
  <div
    class="grid max-w-7xl grid-cols-1 gap-x-8 gap-y-10 px-4 mt-5 sm:px-6 md:grid-cols-3 lg:px-8"
  >
    <nav
      aria-label="Tabs"
      class="flex space-x-4"
    >
      <a
        aria-current="page"
        class="bg-teal-100/50 text-white outline outline-teal-500 px-3 py-0.5 text-sm font-medium"
        href="#"
      >
        Tab 1
      </a>
      <button
        class="text-gray-500 hover:text-gray-700 px-3 py-0.5 text-sm font-medium"
      >
        Tab 2
      </button>
      <a
        class="text-gray-500 hover:text-gray-700 px-3 py-0.5 text-sm font-medium"
        href="#"
      >
        Tab 3
      </a>
    </nav>
  </div>
</div>
`;
